index.html

```diff
@@ -193,5 +193,44 @@ <h2 class="heading">About Me</h2>
                 <button type="button">Get My Resume</button>
             </div>
         </section>
+
+        <section class="language">
+            <ul>
+                <li class="coding-language">
+                    <img src="./images/language.svg" alt="language-icon">
+                    <h2 class="one">Language</h2>
+                    <ul class="stack-btn">
+                        <li class="list">JavaScript</li>
+                        <li class="list">Ruby</li>
+                        <li class="list">Html</li>
+                        <li class="list">css</li>
+                    </ul>
+                </li>
+
+                <li class="coding-language">
+                    <img src="./images/framework.svg" alt="framework-icon">
+                    <h2 class="one">Frameworks</h2>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul` tag, each of them in a `li` tag for best semantic practices.

Reply:
This issue is resolved.